app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -376,23 +402,23 @@ public void onEventMainThread(DownloadEvent event) {
     private final MenuItemUtils.UpdateRefreshMenuItemChecker updateRefreshMenuItemChecker =
             () -> DownloadService.isRunning && DownloadRequester.getInstance().isDownloadingFeeds();
 
-    private final SubscriptionsAdapter.ItemAccess itemAccess = new SubscriptionsAdapter.ItemAccess() {
-        @Override
-        public int getCount() {
-            if (listItems != null) {
-                return listItems.size();
-            } else {
-                return 0;
-            }
-        }
+    @Override
+    public void onEndSelectMode() {
+        speedDialView.close();
+        speedDialView.setVisibility(View.GONE);
+        subscriptionAdapter.setItems(listItems);
+        subscriptionAdapter.notifyDataSetChanged();
+    }
 
-        @Override
-        public NavDrawerData.DrawerItem getItem(int position) {
-            if (listItems != null && 0 <= position && position < listItems.size()) {
-                return listItems.get(position);
-            } else {
-                return null;
+    @Override
+    public void onStartSelectMode() {
+        List<NavDrawerData.DrawerItem> feedsOnly = new ArrayList<>();
```

Comment:
Hmm. The point of the new multi-select feature was that the screen does not change as much when entering. I think I would still display the folders.

Reply:
But the reason I removed them was that when select all is pressed the folders get included even though they shouldn't be. Will we be able to remove the folders once #5318 is implemented? I will be glad to volunteer to work on it.

Reply:
Okay, let's keep it as it is.